### PR TITLE
feat: SYG-484 Integrate with Sygna Bridge

### PR DESCRIPTION
**Pull request by kunmingLiu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +17,7 @@ What we provide:
 * Networking layers
 * [Protocol Buffers][4] wiring and definitions
 * Key signing with [EC cryptography][5]
+* Transfer data with a originator VASP in [Sygna Bridge][8]
```
Comment (delbinwen) [inline README.md]: suggest `data from originator VASP...`

Comment (kunmingLiu) [reply README.md]: fixed

File: src/main/java/com/netki/util/MessagesExtensions.kt [hunk 2]
```diff
@@ -1255,3 +1281,36 @@ internal fun ByteArray.changeStatus(statusCode: StatusCode, statusMessage: Strin
...
+internal fun String.hexStringToByteString(): ByteString {
+    val byteArray = ByteArray(this.length / 2) {
+        this.substring(it * 2, it * 2 + 2).toInt(16).toByte()
+    }
+    return byteArray.toByteString()
+}
+
+/**
+ * Validate that a sygnaParameters is valid.
+ * Is valid, when it has one single primaryOwner.
```
Comment (s60912frank) [inline src/main/java/com/netki/util/MessagesExtensions.kt]: * Validate if sygnaParameters is valid.
* It's valid when it has only one primaryOwner.

**Review by s60912frank — CHANGES_REQUESTED** [review]
comment小修改

File: src/main/java/com/netki/util/MessagesExtensions.kt (hunk 2 above)
Comment (kunmingLiu) [reply src/main/java/com/netki/util/MessagesExtensions.kt]: fixed

**Review by s60912frank — APPROVED** [review]

**Review by Dorac — APPROVED** [review]

**Pull request merged**